File: backend/member-service/src/main/java/com/example/memberservice/common/dto/CommonDataDto.java
package com.example.memberservice.common.dto;

import com.example.memberservice.common.entity.CommonData;

/** 안드로이드가 기대하는 모양 그대로다: {"key":"version","value":"1.2.3"}. */
public record CommonDataDto(String key, String value) {

    public static CommonDataDto from(CommonData commonData) {
        return new CommonDataDto(commonData.getKey(), commonData.getValue());
    }
}

File: android/modu-chat/app/src/main/kotlin/com/example/modumessenger/data/dto/CommonDataDto.kt
package com.example.modumessenger.data.dto

/** `member-service/api-public/common/{key}` 응답. `{"key":"version","value":"1.2.3"}` 모양. */
data class CommonDataDto(
    val key: String? = null,
    val value: String? = null,
)
